extension.js

```diff
@@ -166,6 +170,11 @@ function activate(context) {
     )
   register('evaluateSelection')
   register('executeRegisteredCommand')
+
+  let initialization = () => {
+    E.node_path = E.shell(conf("npmCommand") + " root -g")
+  }
+  initialization()
```

Comment:
Why make the intlization code into a top-level function, only to invoke it immediately?

Why not just invoke it immediately? I.e., inline line 177 with the definition on line 175?

Or do you anticipate it can be used as a hooking mechanism? If so, could you elaborate on your vision here 😁

Reply:
Since under windows it's relatively expensive to start a new process, hence here's i just cache the result of the npm module path to make evaluation faster.